app/src/main/java/de/westnordost/streetcomplete/quests/building_underground/IsBuildingUnderground.java

```diff
@@ -0,0 +1,51 @@
+package de.westnordost.streetcomplete.quests.building_underground;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class IsBuildingUnderground extends SimpleOverpassQuestType
+{
+	@Inject public IsBuildingUnderground(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "ways, relations with building and !location and layer~-.*";
```

Comment:
Did you test whether the tag filter syntax is properly translated to valid overpass? Because of `layer~-.*`, it would be better to test it once.

Reply:
It found expected buildings in Vienna, including one where I was able to make the edit ( https://www.openstreetmap.org/changeset/64655345#map=19/48.20834/16.37226&layers=N ).

I was not checking generated code of overpass turbo query.